### DISCUSSION TITLE
OD can affect star rating of some beatmap?

### DISCUSSION THREAD

**Discussion by DarkbitNetwork** [ppy/osu#25201] [Q&A] [answered]
So I decided to build master branch and use difficulty adjust mod on a [this map](https://osu.ppy.sh/beatmapsets/1151761#osu/2442609) to make the timing window tighter but I discovered that changing OD can heavily affect the star rating of a certain map, some maps are less affected than others though.

Edit: Found out that changing OD to make it lower can make it affect more beatmaps (od0 affects [Giga-P - Electric Angel](https://osu.ppy.sh/beatmapsets/172900#osu/417897), [Frums - Credits](https://osu.ppy.sh/beatmapsets/1372002#osu/3044370), [Lime - Smiling](https://osu.ppy.sh/beatmapsets/1837955#osu/3778695), and probably more)

https://github.com/ppy/osu/assets/43719333/832f9c2b-21a0-4705-9d22-45a7ae745cbc

**Am I forgetting things or is this something new?** 
If I'm forgetting things and this isn't new then it'll be nice to know when this has become the case.
Otherwise, I'd like to know if this is the intended feature.

**Comment by Givikap120** [comment, marked as answer]
There are "doubletapness" in speed pp that can decrease SR if OD is too low
